src/Education/UI/SeverityImages.xaml

```diff
@@ -0,0 +1,70 @@
+﻿<ResourceDictionary
```

Comment:
I added a new `UI` folder for the XAML files. We should move the user control into this folder in a future commit (currently it is in `Commands`).